src/agent/src/util.rs

```diff
@@ -159,6 +165,7 @@ mod tests {
     }
 
     impl ToString for BufWriter {
+        // No way to return an error here, hence the unwrap calls ;(
```

Comment:
We're inside test code, so asserts are standard practice, and `unwrap()` is just a specialized assert.  I don't think this needs commenting on.